scalableminds/webknossos
# Type annotations and Scala Style

Follow scala style guide as suggested by IntelliJ (in most places)
Mostly 
 - making things private that can be
 - give type annotations to all public members
 - remove redundant braces
 - remove some unused variables

There are a few items remaining (datasource package object, type parameter shadowing in isosurfaceservice, as well as a few unused parameters dictated by superclasses) but those are specific and I didn’t want to touch them now, for fear of breaking something. I think we’re already pretty good with this :)

Note that many line changes in the diff are caused by different line-breaks and indentations the formatter introduced. Sorry about that :see_no_evil: 

### URL of deployed dev instance (used for testing):
- https://typeannotations.webknossos.xyz

### Steps to test:
- CI should run through
- wK should start normally, trace some, data should load, annotation should save

### Issues:
- fixes #3565

------
